packages/jest-snapshot/src/Snapshot.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+const fs = require('fs');
+
+class Snapshot {
```

Comment:
Let's call this "TestSnapshot" to make sure it links to the "Test". This is not actually about one single snapshot, but about the snapshot file associated with one test file.